content/doc/book/pipeline/jenkinsfile.adoc

```diff
@@ -25,18 +27,18 @@ provides a number of immediate benefits:
   footnote:[https://en.wikipedia.org/wiki/Single_Source_of_Truth]
   for the Pipeline, which can be viewed and edited by multiple members of the project.
 
-Pipeline supports <<syntax#, two syntaxes>>, Declarative (introduced in
-Pipeline 2.5) and Scripted Pipeline. Both of which support building continuous
+Pipeline supports <<syntax#, two syntaxes>>, declarative (introduced in
```

Comment:
If we want to change the case of these it should be a separate PR and done everywhere.

Reply:
Now reverted